src/modules/colorPicker/ColorPickerUI/Mouse/IMouseInfoProvider.cs

```diff
@@ -18,6 +18,8 @@ public interface IMouseInfoProvider
 
         event MouseUpEventHandler OnMouseDown;
 
+        event RMouseUpEventHandler OnRMouseDown;
```

Comment:
while the hook is abbreviated and is RMouse, we should pivot the methods and properties in code to be maybe be more aligned with maybe either settings or something.  Secondary would be preferable as that is what is actually in settings. if someone switches buttons in settings, i'd hope this would just flip accordingly.

Reply:
So, I should just swap "R" for "Secondary" everywhere? I'm sorry if I ask annoying questions. This is my first ever Open Source PR and I want to make sure I get it right.